src/CommunityToolkit.Maui.Core/AppBuilderExtensions.shared.cs

```diff
@@ -12,6 +14,10 @@ public static class AppBuilderExtensions
 	/// <returns><see cref="MauiAppBuilder"/> initialized for <see cref="CommunityToolkit.Maui.Core"/></returns>
 	public static MauiAppBuilder UseMauiCommunityToolkitCore(this MauiAppBuilder builder)
 	{
+		builder.ConfigureMauiHandlers(handlers =>
+		{
+			handlers.AddHandler<IDrawingView, DrawingViewHandler>();
```

Comment:
Just curious - should we move `Popup` to here too?

Or should we move this to `CommunityToolkit.Maui.AppBuilderExtensions`?

Or is there a reason one is added in `CommunityToolkit.Maui.Core.AppBuilderExtensions` and the other is added in `CommunityToolkit.Maui.AppBuilderExtensions`?

Reply:
I use interface here, that is why I put it in Core.
Pedro used control class and it is in CommunityToolkit.Maui.
the question is does the Comet need this handler registration. if yes, we should move the popup here, otherwise we should move it to CommunityToolkit.Maui.

Reply:
^@pictos What are your thoughts on moving `IPopup`'s handler registration to `CommunityToolkit.Maui.Core`? 

`Popup`'s current handler registration obviously works great! But could we move it here for future toolkit's to leverage too? No worries if its handler registration needs to stay in `CommunityToolkit.Maui`!